core/src/main/resources/org/apache/brooklyn/core/mgmt/persist/deserializingClassRenames.properties

```diff
@@ -1434,3 +1434,5 @@ brooklyn.test.entity.BlockingEntityImpl
 org.apache.brooklyn.config.ConfigInheritance$None                                : org.apache.brooklyn.config.ConfigInheritance$Legacy$None
 org.apache.brooklyn.config.ConfigInheritance$Always                              : org.apache.brooklyn.config.ConfigInheritance$Legacy$Always
 org.apache.brooklyn.config.ConfigInheritance$Merged                              : org.apache.brooklyn.config.ConfigInheritance$Legacy$Merged
+
+org.apache.brooklyn.entity.software.base.lifecycle.MachineLifecycleEffectorTasks$ProvisioningTaskState : org.apache.brooklyn.core.entity.internal.AttributesInternal$ProvisioningTaskState
```

Comment:
I think it will be useful to have a marker here as well.  Agree it can be done separately. A good time to begin will be post-0.10.0 release. Then can treat all previous entries as added in 0.10.0.